app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserActivity.kt

```diff
@@ -39,6 +39,10 @@ open class ExternalAppBrowserActivity : HomeActivity() {
         components.appStartupTelemetry.onExternalAppBrowserOnCreate(safeIntent, hasSavedInstanceState)
     }
 
+    override fun navigateToBrowserOnColdStart() {
+        // No-op for external app
```

Comment:
👍 good catch. Let's also cover this one in `ExternalAppBorwserActivityTest`. We can simple take our test from above and make sure we never call `openToBrowser`, just in case someone removes this empty override.